bin/oneoff/backfill_data/user_school_infos_backfill

```diff
@@ -18,9 +18,13 @@ User.where.not(school_info: nil).where(user_type: "teacher").find_in_batches(bat
         last_confirmation_date: user.created_at
       )
       num_users_updated += 1
+    rescue Exception => e
+      puts "Error: teacher with user id #{user.id} with school id #{user.school_info.id}"
```

Comment:
Should this be school_info_id? Or it might not matter actually...

Reply:
@bethanyaconnor It think it should be fine but open to suggestions